code/modules/mob/living/carbon/human/human.dm

```diff
@@ -240,16 +244,19 @@
 			if(!I || !L || I.loc != src || !(I in L.embedded_objects)) //no item, no limb, or item is not in limb or in the person anymore
 				return
 			var/time_taken = I.embedded_unsafe_removal_time*I.w_class
-			usr.visible_message("<span class='warning'>[usr] attempts to remove [I] from their [L.name].</span>","<span class='notice'>You attempt to remove [I] from your [L.name]... (It will take [time_taken/10] seconds.)</span>")
+			usr.visible_message("<span class='warning'>[usr] attempts to remove [I] from [usr == src ? "their" : name + "'s"] [L.name].</span>",\
+				                "<span class='notice'>You attempt to remove [I] from [usr == src ? "your" : name + "'s"] [L.name]... (It will take [time_taken/10] seconds.)</span>")

```

Comment:
Please use the pronoun helpers found in [this file](https://github.com/HippieStation/HippieStation13/blob/master/code/__HELPERS/pronouns.dm) to insert \the before `[I]` because it might come out rather badly if we don't do this.